projects/com.oracle.truffle.llvm.nodes.impl/src/com/oracle/truffle/llvm/nodes/impl/func/LLVMCallNode.java

```diff
@@ -292,6 +293,11 @@ public Object execute(VirtualFrame frame) {
                         Object[] arguments = frame.getArguments();
```

Comment:
Thanks for noticing the missing `@ExplodeLoop`! I added an assert for the case where the length of the `arguments` array is not a compilation constant. I will add test cases for indirect function calls of native functions with variadic arguments in a subsequent PR, where such cases could potentially happen.